pyproject.toml

```diff
@@ -29,7 +29,7 @@ dsnparse = "*"
 click = "^8.1"
 rich = "*"
 toml = "^0.10.2"
-sqeleton = "0.0.8"
+sqeleton = { path = "./vendor/sqeleton", develop = true }
```

Comment:
On a side note: I wonder, will this approach also package the files properly into tarballs & wheels and install it in a clean environment that is not git-based (i.e. `pip install data-diff`)?